curriculum/challenges/english/07-scientific-computing-with-python/learn-lambda-functions-by-building-an-expense-tracker/66694bfa435cef241f6f9a68.md

```diff
@@ -0,0 +1,55 @@
+---
+id: 66694bfa435cef241f6f9a68
+title: Step 1
+challengeType: 20
+dashedName: step-1
+---
+
+# --description--
+
+Before starting with the project, you are going to learn about lists.
+
+Different from numbers and strings, a <dfn>list</dfn> is a data type that works as a container for other values:
+
+```py
+example_list = [1, 2, 3, 4]
+empty_list = []
+```
+
+The list is characterized by the square brackets around all the values, and a comma between the values, for example `["A", "happy", "list"]`. If the list does not contain any values, then it is an empty list `[]`.
+
+A list can contain different data types: `[1, "Up", ["Down", "Twice]]`. That includes all possible data types. It can also contain an other list!
```

Comment:
```suggestion
A list can contain different data types: `[1, "Up", ["Down", "Twice]]`. That includes all possible data types. It can also contain another list!
```